src/components/m-dropdown/_template.js

```diff
@@ -3,17 +3,17 @@ import raw from 'bel/raw';
 
 const arrowIcon = '<axa-icon id="angle-bracket-down" classes="m-dropdown__icon"></axa-icon>';
 
-const nativeSelect = ({ items }) => bel`<div class="m-dropdown__select-wrap" tabindex="0">
-    <select class="m-dropdown__select">
+const nativeSelect = ({ items, size }) => bel`<div class="m-dropdown__select-wrap ${size === 'sm' ? 'm-dropdown__select-wrap--small' : ''}" tabindex="0">
```

Comment:
This could also be updated to ${size && `m-dropdown__select-wrap--${size}`}

Reply:
i already did it in the changes im doing :) thanks anyway